### PR TITLE
Add integration tests.

### PR DESCRIPTION
**Pull request by jDramaix**

### CODE REVIEW

**Comment by jDramaix** [comment]
This set of tests should give us more confidence that we don't break anything on next bug fixes/improvement

**Comment by christiangoudreau** [comment]
Build failure? Else LGTM

**Comment by jDramaix** [comment]
yep lowercase java filename... It should be fixed now

File: integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/DefaultCarRenderer.java [hunk 1]
```diff
@@ -17,8 +17,10 @@
 package com.arcbees.chosen.integrationtest.client.domain;
 
 import com.google.gwt.text.shared.AbstractRenderer;
+import com.google.gwt.text.shared.Renderer;
 
 public class DefaultCarRenderer extends AbstractRenderer<CarBrand> {
+    public static final Renderer<CarBrand> RENDERER = new DefaultCarRenderer();
```
Comment (meriouma) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/DefaultCarRenderer.java]: new line after 23

**Comment by meriouma** [comment]
LGTM

**Comment by jDramaix** [comment]
Finally...

**Pull request merged**